audformat/core/column.py

```diff
@@ -36,26 +36,31 @@ def assert_values(
     ):
 
         if is_scalar(values):
-            ok = values in scheme
-        else:
-            if isinstance(values, pd.Series):
-                values = values.values
-            if isinstance(values, np.ndarray):
-                if scheme.is_numeric:
-                    # Support type object with None entries,
-                    # which need to be converted to NaN
-                    # to support min/max
-                    values = values.astype(float)
-                    values = [
-                        np.nanmin(values),
-                        np.nanmax(values),
-                    ]
-            values = list(set(values))
-            ok = all([value in scheme for value in values])
-
-    if not ok:
+            values = [values]
+        elif isinstance(values, pd.Series):
+            values = values.values
+        elif isinstance(values, np.ndarray):
+            if scheme.is_numeric:
+                # Support type object with None entries,
+                # which need to be converted to NaN
+                # to support min/max
+                values = values.astype(float)
+                values = [
+                    np.nanmin(values),
+                    np.nanmax(values),
+                ]
+        # Get unique values and preserve order
+        values = list(dict.fromkeys(values))
+        bad_values = [value for value in values if value not in scheme]
+
+    if len(bad_values) > 0:
+        values = str(bad_values[:50])[1:-1]
```

Comment:
I suggest we restrict to 10 items as we also do in https://github.com/audeering/audformat/blob/7c22a3e026e11e58466be6f530ebec4dc790ef34/audformat/core/utils.py#L237 and https://github.com/audeering/audformat/blob/7c22a3e026e11e58466be6f530ebec4dc790ef34/audformat/core/index.py#L139

Reply:
I introduced a `max_display` and set it to 10, which I also think will be easier to read.